extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -74,4 +74,43 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 
 		return vscode.Uri.file(projectFile);
 	}
+
+	/**
+	 * Gets the supported project types
+	 */
+	get projectActions(): (dataworkspace.IProjectAction | dataworkspace.IProjectActionGroup)[] {
+		const action1: dataworkspace.IProjectAction = {
+			id: constants.addItemAction,
+			icon: IconPathHelper.add,
+			run: async (treeItem: dataworkspace.WorkspaceTreeItem): Promise<any> => await this.projectController.addItemPromptFromNode(treeItem)
```

Comment:
For these single line functions you can just return the promise directly - no reason to await if you aren't handling errors or doing anything else. It's just adding extra overhead since it still has to wrap the return value in a promise anyways.

Reply:
Makes sense. Fixed it.